q-pup_config/config.json

```diff
@@ -0,0 +1,211 @@
+{
+    "robot_name": "q-pup",
+    "default_urdf": "True",
+    "urdf_path": "$(find q-pup_config)/urdf/quadruped.urdf",
```

Comment:
Package should be `q_pup_config`, not `q-pup_config`.

Alternatively, I think `qpup_*` is better lol it's already becoming a pain typing that underscore. @wmmc88 thoughts?

Reply:
lol i share the sentiment but its too late xd

Reply:
It's never too late >:)

Reply:
I left it as q_pup for now, lmk if you want to do yours before

Reply:
I'll change it tomorrow after both these PRs are merged